src/Data/Reader/Filter/All.php

```diff
@@ -2,11 +2,11 @@
 
 declare(strict_types=1);
 
-namespace Yiisoft\Yii\Cycle\Data\Reader\Processor;
+namespace Yiisoft\Yii\Cycle\Data\Reader\Filter;
 
 use Cycle\ORM\Select\QueryBuilder;
 
-final class All extends GroupProcessor
+final class All extends Group
```

Comment:
```suggestion
final class AllHandler extends Group
```

For consistency with https://github.com/yiisoft/data/blob/master/src/Reader/Iterable/FilterHandler/AllHandler.php

And same to other filter handlers.

Reply:
Is a Iterable handler, but I rename simple filter